crates/js-sys/src/lib.rs

```diff
@@ -314,6 +314,13 @@ extern "C" {
     #[wasm_bindgen(method)]
     pub fn sort(this: &Array) -> Array;
 
+    /// The splice() method changes the contents of an array by removing existing elements and/or
+    /// adding new elements.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
```

Comment:
I think this may be a copy/paste error at the end with `slice` instead of `splice`?

Reply:
:+1: